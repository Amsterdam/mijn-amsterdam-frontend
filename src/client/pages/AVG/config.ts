import { AVGRequestFrontend } from '../../../server/services/avg/types';
import { dateSort } from '../../../universal/helpers/date';
import { LinkProps } from '../../../universal/types';
import {
  DisplayProps,
  WithDetailLinkComponent,
} from '../../components/Table/TableV2';

export const displayPropsAanvragen: DisplayProps<
  WithDetailLinkComponent<AVGRequestFrontend>
> = {
  detailLinkComponent: 'Nummer',
  ontvangstDatumFormatted: 'Ontvangen op',
  themas: 'Onderwerp(en)',
};

export const listPageParamKind = {
  inProgress: 'lopende-aanvragen',
  completed: 'afgehandelde-aanvragen',
};

const tableConfigBase = {
  sort: dateSort('dateRequest', 'desc'),
  displayProps: displayPropsAanvragen,
};

export const tableConfig = {
  [listPageParamKind.inProgress]: {
    title: 'Lopende aanvragen',
    filter: (avgVerzoek: AVGRequestFrontend) => !avgVerzoek.datumAfhandeling,
    ...tableConfigBase,
  },
  [listPageParamKind.completed]: {
    title: 'Afgehandelde aanvragen',
    filter: (avgVerzoek: AVGRequestFrontend) => avgVerzoek.datumAfhandeling,
    ...tableConfigBase,
  },
};

export const linkListItems: LinkProps[] = [
  {
    to: 'https://www.amsterdam.nl/privacy/loket/',
    title: 'Loket persoonsgegevens gemeente Amsterdam',
  },
];
