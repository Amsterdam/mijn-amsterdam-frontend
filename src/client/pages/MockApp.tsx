import { RecoilRoot, RecoilRootProps } from 'recoil';
import React from 'react';
import { MemoryRouter, Route } from 'react-router-dom';

interface MockAppProps {
  routePath: string;
  routeEntry: string;
  initializeState?: RecoilRootProps['initializeState'];
  component: React.Component | any;
}

export default function MockApp({
  routePath,
  routeEntry,
  initializeState,
  component,
}: MockAppProps) {
  return (
    <RecoilRoot initializeState={initializeState}>
      <MemoryRouter initialEntries={[routeEntry]}>
        <Route path={routePath} component={component} />
      </MemoryRouter>
    </RecoilRoot>
  );
}
