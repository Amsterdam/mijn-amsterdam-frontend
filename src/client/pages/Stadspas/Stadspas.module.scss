@import '../../styles/_global.scss';

.Stadspas {
  @include select(PageContent) {
    padding-bottom: $page-content-bottom-space;
  }
}

.Table {
  tr {
    td:nth-child(1) {
      width: 20%;
    }
    td:nth-child(2) {
      width: 30%;
    }
    td:nth-child(3) {
      width: 30%;
    }
    @include select(Table_DisplayPropTitle) {
      &:nth-child(1) {
        width: 50%;
      }
    }
  }
}

.SectionCollapsibleFirst {
  border-top: 0.1rem solid $color-neutral-grey2;
  padding-top: 1.7rem;
}
