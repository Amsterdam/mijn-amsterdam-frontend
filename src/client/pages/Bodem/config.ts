import { generatePath } from 'react-router-dom';

import { LoodMetingFrontend } from '../../../server/services/bodem/types';
import { AppRoutes } from '../../../universal/config/routes';
import { dateSort } from '../../../universal/helpers/date';
import { LinkProps } from '../../../universal/types';

const displayPropsAanvragen = {
  detailLinkComponent: 'Adres',
  datumAanvraagFormatted: 'Aangevraagd',
  status: 'Status',
};

const listPageParamKind = {
  inProgress: 'lopende-aanvragen',
  completed: 'afgehandelde-aanvragen',
};

const tableConfigBase = {
  sort: dateSort('dateRequest', 'desc'),
  displayProps: displayPropsAanvragen,
};

export const tableConfig = {
  [listPageParamKind.inProgress]: {
    ...tableConfigBase,
    title: 'Lopende aanvragen',
    filter: (bodemAanvraag: LoodMetingFrontend) => !bodemAanvraag.processed,
    listPageRoute: generatePath(AppRoutes['BODEM/LIST'], {
      kind: listPageParamKind.inProgress,
    }),
  },
  [listPageParamKind.completed]: {
    ...tableConfigBase,
    title: 'Afgehandelde aanvragen',
    filter: (bodemAanvraag: LoodMetingFrontend) => bodemAanvraag.processed,
    listPageRoute: generatePath(AppRoutes['BODEM/LIST'], {
      kind: listPageParamKind.completed,
    }),
  },
} as const;

export const linkListItems: LinkProps[] = [
  {
    title: 'Meer informatie over lood in de bodem.',
    to: 'https://www.amsterdam.nl/wonen-leefomgeving/bodem/lood-grond/',
  },
] as const;
