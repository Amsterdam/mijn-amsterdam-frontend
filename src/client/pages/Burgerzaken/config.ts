import { generatePath } from 'react-router-dom';

import { AppRoutes } from '../../../universal/config/routes';
import { dateSort } from '../../../universal/helpers/date';
import { IdentiteitsbewijsFrontend, LinkProps } from '../../../universal/types';
import { withOmitDisplayPropsForSmallScreens } from '../../components/Table/helpers';
import {
  DisplayProps,
  WithDetailLinkComponent,
} from '../../components/Table/TableV2';

const displayPropsBase: DisplayProps<
  WithDetailLinkComponent<IdentiteitsbewijsFrontend>
> = {
  detailLinkComponent: 'Type',
  datumAfloopFormatted: 'Geldig tot',
};

const listPageParamKind = {
  identiteitsbewijzen: 'identiteitsbewijzen',
};

const displayProps = withOmitDisplayPropsForSmallScreens(displayPropsBase, [
  'datumAfloopFormatted',
]);

export const tableConfig = {
  [listPageParamKind.identiteitsbewijzen]: {
    title: 'Mijn reisdocumenten',
    displayProps,
    sort: dateSort('datumAfloop', 'desc'),
    listPageRoute: generatePath(AppRoutes['BURGERZAKEN/LIST'], {
      kind: listPageParamKind.identiteitsbewijzen,
    }),
  },
} as const;

export const linkListItems: LinkProps[] = [
  {
    to: 'https://www.amsterdam.nl/burgerzaken',
    title: 'Overzicht en aanvragen bij burgerzaken',
  },
] as const;

export const backLinkDetailPage = AppRoutes.BURGERZAKEN;
