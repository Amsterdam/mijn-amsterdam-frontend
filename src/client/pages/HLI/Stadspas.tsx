import { useMemo } from 'react';
import { generatePath } from 'react-router-dom';
import { Stadspas } from '../../../server/services/hli/stadspas-types';
import { AppRoutes } from '../../../universal/config';
import { ThemaTitles } from '../../config/thema';
import { isError, isLoading } from '../../../universal/helpers';
import { defaultDateFormat } from '../../../universal/helpers/date';
import {
  ThemaIcon,
  ErrorAlert,
  Linkd,
  MaintenanceNotifications,
  OverviewPage,
  PageContent,
  PageHeading,
  SectionCollapsible,
  Table,
} from '../../components';
import { LinkdInline } from '../../components/Button/Button';
import { ExternalUrls } from '../../config/app';
import { useAppStateGetter } from '../../hooks/useAppState';
import styles from './Stadspas.module.scss';

const stadspasDisplayProps = {
  owner: '',
  passNumber: 'Stadspasnummer',
  displayDateEnd: 'Einddatum',
  detailPageUrl: 'Tegoed',
};

function hasMultipleOwners(stadspassen: Stadspas[] | undefined) {
  if (stadspassen === undefined) {
    return false;
  }

  return stadspassen.some((pas) => pas.owner !== stadspassen[0].owner);
}

export default function CStadspas() {
  const { STADSPAS } = useAppStateGetter();
  const aanvragen = STADSPAS.content?.aanvragen;
  const hasStadspas = !!STADSPAS?.content?.stadspassen?.length;

  const stadspasItems = useMemo(() => {
    if (!STADSPAS.content?.stadspassen) {
      return [];
    }
    return STADSPAS.content.stadspassen.map((stadspas) => {
      return {
        ...stadspas,
        displayDateEnd: defaultDateFormat(stadspas.dateEnd),
        detailPageUrl: !!stadspas.budgets.length && (
          <LinkdInline
            href={generatePath(AppRoutes['STADSPAS/SALDO'], {
              id: stadspas.id,
            })}
          >
            Bekijk saldo
          </LinkdInline>
        ),
        owner: stadspas.owner,
      };
    });
  }, [STADSPAS.content]);

  const isLoadingStadspas = isLoading(STADSPAS);

  return (
    <OverviewPage className={styles.Stadspas}>
      <PageHeading
        backLink={{
          to: AppRoutes.HOME,
          title: 'Home',
        }}
        isLoading={false}
        icon={<ThemaIcon />}
      >
        {ThemaTitles.STADSPAS}
      </PageHeading>
      <PageContent>
        <h4>Hoe weet ik of ik een geldige Stadspas heb?</h4>
        <p>
          Hieronder staat het Stadspasnummer van uw actieve pas. Dit pasnummer
          staat ook op de achterkant van uw pas.
        </p>
        <p>
          Bel <a href="tel:020 252 6000">020 252 6000</a> om een nieuwe Stadspas
          aan te vragen.
        </p>
        {!isLoadingStadspas &&
          hasMultipleOwners(STADSPAS.content?.stadspassen) && (
            <p>
              Hebt u kinderen of een partner met een Stadspas? Dan ziet u
              hieronder ook hun Stadspassen.
            </p>
          )}
        <p>
          <Linkd external={true} href={ExternalUrls.STADSPAS}>
            Meer informatie over de Stadspas
          </Linkd>
        </p>
        <MaintenanceNotifications page="stadspas" />
        {isError(STADSPAS) && (
          <ErrorAlert>
            We kunnen op dit moment niet alle gegevens tonen.
          </ErrorAlert>
        )}
      </PageContent>

      {hasStadspas && (
        <SectionCollapsible
          id="SectionCollapsible-stadpas"
          title="Stadspas"
          startCollapsed={false}
          isLoading={isLoadingStadspas}
          hasItems={hasStadspas}
          className={styles.SectionCollapsibleFirst}
        >
          <Table
            items={stadspasItems}
            displayProps={stadspasDisplayProps}
            className={styles.Table}
          />
        </SectionCollapsible>
      )}
    </OverviewPage>
  );
}
