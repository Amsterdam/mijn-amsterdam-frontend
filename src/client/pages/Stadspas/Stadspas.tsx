import { useMemo } from 'react';
import { generatePath } from 'react-router-dom';
import type { FocusItem } from '../../../server/services/focus/focus-types';
import { AppRoutes, ChapterTitles } from '../../../universal/config';
import { dateSort, isError, isLoading } from '../../../universal/helpers';
import { defaultDateFormat } from '../../../universal/helpers/date';
import {
  addTitleLinkComponent,
  Alert,
  ChapterIcon,
  Linkd,
  OverviewPage,
  PageContent,
  PageHeading,
  SectionCollapsible,
  Table,
  MaintenanceNotifications,
} from '../../components';
import { LinkdInline } from '../../components/Button/Button';
import { ExternalUrls } from '../../config/app';
import { useAppStateGetter } from '../../hooks/useAppState';
import styles from './Stadspas.module.scss';

const stadspasDisplayProps = {
  naam: '',
  pasnummer: 'Stadspasnummer',
  displayDatumAfloop: 'Einddatum',
  detailPageUrl: 'Tegoed',
};

const requestsDisplayProps = {
  displayDateStart: 'Datum aanvraag',
  status: 'Status',
};

const decisionsDisplayProps = {
  displayDateStart: 'Datum aanvraag',
  displayDateEnd: 'Datum besluit',
};

export default function Stadspas() {
  const { FOCUS_AANVRAGEN, FOCUS_STADSPAS } = useAppStateGetter();
  const aanvragen = FOCUS_AANVRAGEN.content;

  const items: FocusItem[] = useMemo(() => {
    if (!aanvragen) {
      return [];
    }
    return addTitleLinkComponent(
      aanvragen
        .filter((aanvraag) => aanvraag.productTitle === 'Stadspas')
        .map((item) => {
          return Object.assign({}, item, {
            displayDateEnd: defaultDateFormat(
              item.dateEnd || item.datePublished
            ),
            displayDateStart: defaultDateFormat(item.dateStart),
          });
        })
        .sort(dateSort('datePublished', 'desc'))
    );
  }, [aanvragen]);

  const itemsRequested = items.filter((item) => item.status !== 'Besluit');
  const itemsDecided = items.filter((item) => item.status === 'Besluit');

  const hasActiveRequests = !!itemsRequested.length;
  const hasActiveDescisions = !!itemsDecided.length;
  const hasStadspas = !!FOCUS_STADSPAS?.content?.stadspassen?.length;

  const stadspasItems = useMemo(() => {
    if (!FOCUS_STADSPAS.content?.stadspassen) {
      return [];
    }
    return FOCUS_STADSPAS.content.stadspassen.map((stadspas) => {
      return {
        ...stadspas,
        displayDatumAfloop: defaultDateFormat(stadspas.datumAfloop),
        detailPageUrl: !!stadspas.budgets.length && (
          <LinkdInline
            href={generatePath(AppRoutes['STADSPAS/SALDO'], {
              id: stadspas.id,
            })}
          >
            Bekijk saldo
          </LinkdInline>
        ),
      };
    });
  }, [FOCUS_STADSPAS.content]);

  const isLoadingFocus = isLoading(FOCUS_AANVRAGEN);
  const isLoadingStadspas = isLoading(FOCUS_STADSPAS);

  return (
    <OverviewPage className={styles.Stadspas}>
      <PageHeading
        backLink={{
          to: AppRoutes.HOME,
          title: 'Home',
        }}
        isLoading={false}
        icon={<ChapterIcon />}
      >
        {ChapterTitles.STADSPAS}
      </PageHeading>
      <PageContent>
        <p>Hieronder vindt u meer informatie over uw eigen Stadspas.</p>
        {!isLoadingStadspas && FOCUS_STADSPAS.content?.type !== 'kind' && (
          <p>
            Hebt u kinderen of een partner met een Stadspas? Dan ziet u
            hieronder ook hun Stadspassen.
          </p>
        )}
        <p>
          <Linkd external={true} href={ExternalUrls.STADSPAS}>
            Meer informatie over de Stadspas
          </Linkd>
        </p>
        <MaintenanceNotifications page="stadspas" />
        {(isError(FOCUS_AANVRAGEN) || isError(FOCUS_STADSPAS)) && (
          <Alert type="warning">
            <p>We kunnen op dit moment niet alle gegevens tonen.</p>
          </Alert>
        )}
      </PageContent>

      {hasStadspas && (
        <SectionCollapsible
          id="SectionCollapsible-stadpas"
          title="Stadspas"
          startCollapsed={false}
          isLoading={isLoadingStadspas}
          hasItems={hasStadspas}
          track={{
            category: 'Stadspas overzicht / Stadpas',
            name: 'Datatabel',
          }}
          className={styles.SectionCollapsibleFirst}
        >
          <Table
            titleKey="moreDetail"
            items={stadspasItems}
            displayProps={stadspasDisplayProps}
            className={styles.Table}
          />
        </SectionCollapsible>
      )}

      <SectionCollapsible
        id="SectionCollapsible-stadspas-request-process"
        title="Lopende aanvragen"
        startCollapsed={hasStadspas}
        isLoading={isLoadingFocus}
        hasItems={hasActiveRequests}
        track={{
          category: 'Stadspas overzicht / Lopende aanvragen',
          name: 'Datatabel',
        }}
        noItemsMessage="U hebt op dit moment geen lopende aanvragen."
        className={styles.SectionCollapsible}
      >
        <Table
          items={itemsRequested}
          displayProps={requestsDisplayProps}
          className={styles.Table}
        />
      </SectionCollapsible>

      <SectionCollapsible
        id="SectionCollapsible-stadspas-request-process-decisions"
        startCollapsed={true}
        isLoading={isLoadingFocus}
        hasItems={hasActiveDescisions}
        title="Eerdere aanvragen"
        track={{
          category: 'Stadspas overzicht / Eerdere aanvragen',
          name: 'Datatabel',
        }}
        noItemsMessage="U hebt op dit moment geen eerdere aanvragen."
      >
        <Table
          items={itemsDecided}
          displayProps={decisionsDisplayProps}
          className={styles.Table}
        />
      </SectionCollapsible>
    </OverviewPage>
  );
}
