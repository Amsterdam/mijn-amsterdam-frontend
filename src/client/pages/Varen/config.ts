import { generatePath } from 'react-router-dom';

import { AppRoutes } from '../../../universal/config/routes';
import { dateSort } from '../../../universal/helpers/date';
import { VarenFrontend } from '../../../server/services/varen/config-and-types';

export const displayPropsAanvragen = {
  detailLinkComponent: '',
  title: 'Omschrijving',
  dateRequestFormatted: 'Aangevraagd',
  status: 'Status',
};

export const listPageParamKind = {
  inProgress: 'lopende-aanvragen',
  completed: 'afgehandelde-aanvragen',
};

const tableConfigBase = {
  sort: dateSort('dateRequest', 'desc'),
  displayProps: displayPropsAanvragen,
};

export const tableConfig = {
  [listPageParamKind.inProgress]: {
    title: 'Lopende aanvragen',
    filter: (vergunning: VarenFrontend) => !vergunning.dateEnd,
    listPageRoute: generatePath(AppRoutes['VAREN/LIST'], {
      kind: listPageParamKind.inProgress,
    }),
    ...tableConfigBase,
  },
  [listPageParamKind.completed]: {
    title: 'Afgehandelde aanvragen',
    filter: (vergunning: VarenFrontend) => vergunning.dateEnd,
    listPageRoute: generatePath(AppRoutes['VAREN/LIST'], {
      kind: listPageParamKind.completed,
    }),
    ...tableConfigBase,
  },
};
