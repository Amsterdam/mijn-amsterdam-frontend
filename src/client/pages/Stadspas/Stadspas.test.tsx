import { render } from '@testing-library/react';
import { generatePath } from 'react-router-dom';
import { MutableSnapshot } from 'recoil';
import { AppRoutes } from '../../../universal/config';
import { appStateAtom } from '../../hooks/useAppState';
import MockApp from '../MockApp';
import Stadspas from './Stadspas';

const testState: any = {
  WPI_STADSPAS: {
    content: {
      adminNumber: '123123123',
      ownerType: 'hoofdpashouder',
      stadspassen: [
        {
          budgets: [
            {
              title: 'SPORT-EN-SPEL',
              assigned: 220,
              balance: 80,
              urlTransactions:
                '/wpi/stadspastransacties/gAAAAABfmojHaIr3lIb5ATsZ98is4S8x3HqxdbVACw8562VCtv3ygKtkD5h7rGCUZhoebm0jFAvLjzibxcgKqpbWELopLBT5Ywf1FzDjzvNVQOij_3hYLks=',
            },
          ],
          dateEnd: '2021-08-31T21:59:59.000Z',
          id: '200769',
          owner: 'G Braber',
          passType: 'kind',
          passNumber: '6011012604273',
        },
      ],
      aanvragen: [
        {
          title: 'Aanvraag stadspas',
          datePublished: '2020-07-24',
          dateStart: '2020-07-14',
          statusId: 'besluit',
          steps: [
            {
              id: 'besluit',
              status: 'Besluit',
              documents: [],
              datePublished: '2020-09-28T00:00:00+02:00',
              decision: 'afwijzing',
            },
          ],
        },
        {
          title: 'Aanvraag stadspas 2',
          datePublished: '2020-07-24',
          dateStart: '2020-07-14',
          statusId: 'herstelTermijn',
          steps: [
            {
              id: 'herstelTermijn',
              status: 'Meer informatie',
              documents: [],
              datePublished: '2020-08-28T00:00:00+02:00',
              decision: 'afwijzing',
            },
          ],
        },
      ],
    },
    status: 'OK',
  },
};

function initializeState(snapshot: MutableSnapshot) {
  snapshot.set(appStateAtom, testState);
}

describe('<Stadpas />', () => {
  const routeEntry = generatePath(AppRoutes.STADSPAS);
  const routePath = AppRoutes.STADSPAS;

  const Component = () => (
    <MockApp
      routeEntry={routeEntry}
      routePath={routePath}
      component={Stadspas}
      initializeState={initializeState}
    />
  );

  it('Matches the Full Page snapshot', () => {
    const { asFragment } = render(<Component />);
    expect(asFragment()).toMatchSnapshot();
  });
});
