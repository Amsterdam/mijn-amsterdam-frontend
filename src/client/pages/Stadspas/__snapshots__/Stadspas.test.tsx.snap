// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Stadpas /> Matches the Full Page snapshot 1`] = `
<DocumentFragment>
  <main
    class="Page OverviewPage Stadspas"
  >
    <header
      class="PageHeading hasIcon hasBackLink"
    >
      <span
        aria-hidden="true"
        class="Icon"
      >
        <svg
          aria-label="Stadspas"
          class="ChapterIcon"
          fill="black"
        >
          stadspas.svg
        </svg>
      </span>
      <div
        class="HeadingInner"
      >
        <a
          class="Button Button__plain Button__lean Linkd BackLink"
          href="/"
        >
          <svg
            aria-hidden="true"
            class="Icon Icon__left"
            fill="#000"
            height="14px"
            width="14px"
          >
            Chevron-Left.svg
          </svg>
          Home
        </a>
        <h2
          class="Heading Heading__large"
        >
          Stadspas
        </h2>
      </div>
    </header>
    <div
      class="PageContent"
    >
      <p>
        Hieronder vindt u meer informatie over uw eigen Stadspas.
      </p>
      <p>
        Hebt u kinderen of een partner met een Stadspas? Dan ziet u hieronder ook hun Stadspassen.
      </p>
      <p>
        <a
          class="Button Button__plain Button__lean Linkd"
          href="https://www.amsterdam.nl/stadspas"
          rel="external noopener noreferrer"
        >
          <svg
            aria-hidden="true"
            class="Icon Icon__left"
            fill="#000"
            height="14px"
            width="14px"
          >
            Chevron-Right.svg
          </svg>
          Meer informatie over de Stadspas
        </a>
      </p>
    </div>
    <section
      class="SectionCollapsible SectionCollapsibleFirst"
    >
      <h3
        class="Heading Heading__mediumLarge Title TitleWithItems"
      >
        <button
          aria-expanded="true"
          class="TitleToggle"
        >
          <svg
            aria-hidden="true"
            class="CaretIcon"
          >
            Chevron-Right.svg
          </svg>
           Stadspas
        </button>
      </h3>
      <div
        aria-hidden="false"
        class="Panel"
        style="height: 0px;"
      >
        <div
          class="PanelInner"
        >
          <table
            class="Table Table"
          >
            <thead>
              <tr
                class="TableRow"
              >
                <td
                  class="DisplayProp"
                />
                <th
                  class="DisplayProp"
                >
                  Stadspasnummer
                </th>
                <th
                  class="DisplayProp"
                >
                  Einddatum
                </th>
                <th
                  class="DisplayProp"
                >
                  Tegoed
                </th>
              </tr>
            </thead>
            <tbody>
              <tr
                class="TableRow"
              >
                <td
                  class="DisplayProp"
                >
                  <span>
                    G Braber   (kind)
                  </span>
                </td>
                <td
                  class="DisplayProp"
                >
                  <span
                    class="DisplayPropLabel"
                  >
                    Stadspasnummer:
                  </span>
                  <span>
                    6011012604273
                  </span>
                </td>
                <td
                  class="DisplayProp"
                >
                  <span
                    class="DisplayPropLabel"
                  >
                    Einddatum:
                  </span>
                  <span>
                    31 augustus 2021
                  </span>
                </td>
                <td
                  class="DisplayProp"
                >
                  <span
                    class="DisplayPropLabel"
                  >
                    Tegoed:
                  </span>
                  <a
                    class="Button Button__inline Button__lean Linkd LinkedInline"
                    href="/stadspas/saldo/200769"
                  >
                    Bekijk saldo
                  </a>
                </td>
              </tr>
            </tbody>
          </table>
        </div>
      </div>
    </section>
    <section
      class="SectionCollapsible SectionCollapsible"
    >
      <h3
        class="Heading Heading__mediumLarge Title TitleWithItems"
      >
        <button
          aria-expanded="false"
          class="TitleToggle"
        >
          <svg
            aria-hidden="true"
            class="CaretIcon"
          >
            Chevron-Right.svg
          </svg>
           Lopende aanvragen
        </button>
      </h3>
      <div
        aria-hidden="true"
        class="Panel"
        style="height: 0px;"
      >
        <div
          class="PanelInner"
        >
          <table
            class="Table Table"
          >
            <thead>
              <tr
                class="TableRow"
              >
                <td
                  class="DisplayProp"
                />
                <th
                  class="DisplayProp"
                >
                  Datum aanvraag
                </th>
                <th
                  class="DisplayProp"
                >
                  Status
                </th>
              </tr>
            </thead>
            <tbody>
              <tr
                class="TableRow"
              >
                <td
                  class="DisplayPropTitle"
                >
                  Aanvraag stadspas 2
                </td>
                <td
                  class="DisplayProp"
                >
                  <span
                    class="DisplayPropLabel"
                  >
                    Datum aanvraag:
                  </span>
                  <span>
                    14 juli 2020
                  </span>
                </td>
                <td
                  class="DisplayProp"
                >
                  <span
                    class="DisplayPropLabel"
                  >
                    Status:
                  </span>
                  <span>
                    Meer informatie
                  </span>
                </td>
              </tr>
            </tbody>
          </table>
        </div>
      </div>
    </section>
    <section
      class="SectionCollapsible"
    >
      <h3
        class="Heading Heading__mediumLarge Title TitleWithItems"
      >
        <button
          aria-expanded="false"
          class="TitleToggle"
        >
          <svg
            aria-hidden="true"
            class="CaretIcon"
          >
            Chevron-Right.svg
          </svg>
           Eerdere aanvragen
        </button>
      </h3>
      <div
        aria-hidden="true"
        class="Panel"
        style="height: 0px;"
      >
        <div
          class="PanelInner"
        >
          <table
            class="Table Table"
          >
            <thead>
              <tr
                class="TableRow"
              >
                <td
                  class="DisplayProp"
                />
                <th
                  class="DisplayProp"
                >
                  Datum aanvraag
                </th>
                <th
                  class="DisplayProp"
                >
                  Datum besluit
                </th>
              </tr>
            </thead>
            <tbody>
              <tr
                class="TableRow"
              >
                <td
                  class="DisplayPropTitle"
                >
                  Aanvraag stadspas
                </td>
                <td
                  class="DisplayProp"
                >
                  <span
                    class="DisplayPropLabel"
                  >
                    Datum aanvraag:
                  </span>
                  <span>
                    14 juli 2020
                  </span>
                </td>
                <td
                  class="DisplayProp"
                >
                  <span
                    class="DisplayPropLabel"
                  >
                    Datum besluit:
                  </span>
                  <span>
                    24 juli 2020
                  </span>
                </td>
              </tr>
            </tbody>
          </table>
        </div>
      </div>
    </section>
  </main>
</DocumentFragment>
`;
